heuristics/dbic_bids.py

```diff
@@ -20,6 +20,38 @@ def create_key(subdir, file_suffix, outtype=('nii.gz', 'dicom'),
     return template, outtype, annotation_classes
 
 
+# XXX: hackhackhack
+protocols2fix = ['dbic^pulse_sequences']
```

Comment:
for those with simple remappings (not requiring any custom logic), you could make it a dict here, from study description to those  replace items.  Then inside you could just check if you have an entry in it, and if so -- use it in that replacement loop